spec/operators/combineLatest-spec.js

```diff
@@ -67,4 +70,254 @@ describe('combineLatest', function () {
         done();
       });
   }, 300);
+  
+  // From RxJS 2 below
+  
+  
+  it('should work with two nevers', function () {
+    var e1 = Observable.never(),
+      e2 = Observable.never();
+    
+    expectObservable(e1.combineLatest(e2, function (x, y) { return x + y; })).toBe('-');
+  });
+  
+  it('should work with never and empty', function () {
+    var e1 = Observable.never(),
+      e2 = Observable.empty();
+
+    expectObservable(e1.combineLatest(e2, function (x, y) { return x + y; })).toBe('-');
+  });
+
+  it('should work with empty and never', function () {
+    var e1 = Observable.empty(),
+      e2 = Observable.never();
+
+    expectObservable(e1.combineLatest(e2, function (x, y) { return x + y; })).toBe('-');
+  });
+  
+  it('should work with empty and empty', function () {
+    var e1 = Observable.empty(),
+      e2 = Observable.empty();
+
+    expectObservable(e1.combineLatest(e2, function (x, y) { return x + y; })).toBe('|');
+  });
+
+  it('should work with hot-empty and hot-single', function () {    
+    var values = { 
+      a: 1, 
+      b: 2, 
+      c: 3, 
+      r: 1 + 3 //a + c
+    };
+    var e1 =        hot('-a-^-|', values);
+    var e2 =        hot('-b-^-c-|', values);
+    var expected =         '----|';
```

Comment:
There's a test-helpers.js file in the spec directory under helpers (I think). I touched it in this PR.